hack/update-release-parameters.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# Accept a file path for param yaml.
+releaseParams=$1
+
+# Get the paths to all files that can be automatically updated for a release.
+readarray updates < <(yq e -o=j -I=0 '.updates[]' $releaseParams)
+
+for update in "${updates[@]}"; do
+  targetFile=$(echo $update | yq -e '.targetFile')
+  templatePath=$(echo $update | yq -e '.template')
+  echo -e 'Regenerating...' $targetFile
+  draft=$(cat "$templatePath")
+
+  readarray -d " " replacements < <(echo $(echo $update | yq e '.replacements[]')) 
+  for find in "${replacements[@]}"; do
+    find=$(echo $find | tr -d '\n')
+    replace=$(cat $releaseParams | yq -e ".release-tags.${find}")
+    draft=$(echo "$draft" | sed -e "s|{{$find}}|$replace|")
```

Comment:
📝 Could be alright, but noting that replacing in a loop can... unnest placeholders. e.g. `{{sock}} -> {{shoe}} -> boot`.

🔧 Should use a `g` modifier here to handle lines with multiple placeholders.

```
$ echo '{{sock}} {{sock}}' | sed -e 's|{{sock}}|shoe|'
shoe {{sock}}

$ echo '{{sock}} {{sock}}' | sed -e 's|{{sock}}|shoe|g'
shoe shoe
```

```suggestion
    draft=$(echo "$draft" | sed -e "s|{{$find}}|$replace|g")
```